Cargo.toml

```diff
@@ -52,3 +53,7 @@ required-features = ["std"]
 [[example]]
 name = "xpub_descriptors"
 required-features = ["std"]
+
+[[example]]
+name = "taproot"
```

Comment:
You would also need to add a line for this in test.sh so that it is checked in CI

Reply:
Right, thanks!